README.md

```diff
@@ -388,6 +388,13 @@ A curated list of awesome Java frameworks, libraries and software.
 * [JCTools](https://github.com/JCTools/JCTools) - Concurrency tools currently missing from the JDK.
 * [Koloboke](https://github.com/OpenHFT/Koloboke) - Hash sets and hash maps.
 
+## Hosting
+
+*Cloud hosting services for Java applications.*
```

Comment:
You might also consider [Heroku](https://devcenter.heroku.com/categories/java).

I was initially also going to suggest adding [Google App Engine](https://cloud.google.com/appengine/), but it seems that it's support for Java 8 is in beta, so I hesitate to actually suggest it.